source/operators/frame_extractor/frame_extractor.py

```diff
@@ -0,0 +1,162 @@
+###############################################################################
+# PURPOSE:
+#   Lambda function to extract frames from video files using cv2 and store on S3.
+#   WARNING: This function might needs longer Lambda timeouts depending on chunk 
+#   size and how many frames should be extracted.
+###############################################################################
+
+import cv2
```

Comment:
Can you also include changes to the lambda layer factory (`deployment/lambda_layer_factory`) so that cv2 is included in deployment package?

Reply:
frame extraction is not required for custom labels on image - I think we can handle 2 separate PRs - one for custom labels with images and then separately for frame extraction and custom labels with frames. Is that okay ?
@ianwow

Reply:
I agree. Can you submit a new PR (or modify this one) that includes custom labels Image code but does not include the frame extraction code?